### ISSUE TITLE
Is there pre trained model?

### ISSUE THREAD

**Issue by ghost** [ctwxdd/Tensorflow-ACGAN-Anime-Generation#1]
I am running this code and Epoch is 144.

---------------------------------
Epoch 144, Current_step 102000
Discriminator loss :0.29509973526000977
Generator loss     :68.52684020996094
Cls loss     :0.012327473610639572
Cls g loss     :7.715950965881348
---------------------------------
Epoch 144, Current_step 102100
Discriminator loss :0.25976553559303284
Generator loss     :62.120853424072266
Cls loss     :0.014904022216796875
Cls g loss     :7.294325828552246
---------------------------------
Epoch 144, Current_step 102200
Discriminator loss :0.1719554364681244
Generator loss     :59.89821243286133
Cls loss     :0.01943761669099331
Cls g loss     :6.931403160095215
---------------------------------

But, Cls g loss does not goes down.

Is there any pretrained model?

**Comment by ctwxdd** [comment]
Please try the pretrained model here. Thanks!
https://drive.google.com/file/d/1gzJ1_OxGloWSYtGndZYT42SmRfA6HFAN/view?usp=sharing
